### PR TITLE
New package: GPUTemp.GPUTemp version 1.0

### LINKED ISSUE #191699: [Package Request]: GPU Temp
### How can we help?

I would like someone else to build the manifest.

### Please read and ensure the following

- [x] The installer meets the above requirements

### Please provide the following information

Download Page Url: https://www.gputemp.com/
Publisher: softsea
Package Name: GPU Temp
Description: GPU Temp is a free GPU temperature monitor that can display GPU core temperature and load, the temperature data will display in the system tray, and will be real-time updated. In addition, you can choose the color for temperature display, also supports start with the Windows system.
Package Version:  1.0
Installer Url:  https://file1.softsea.com/download/bd5cefd4/9be173058eaa384fad93a22d9f07fff9/Diagnostic_Software/gputemp_setup.exe

### PR DESCRIPTION
**Pull request by Dragon1573**
## Checklist for Pull Requests

- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Resolve #191699

## Manifests

- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/doc/Authoring.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.6 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.6.0)?

Note: `<path>` is the directory's name containing the manifest you're submitting.

## Manual validation

image
image

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/191713)

### CODE REVIEW

File: manifests/g/GPUTemp/GPUTemp/1.0/GPUTemp.GPUTemp.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2019-10-01
+ProductCode: '{8C8711FD-0FC8-4801-B33E-ED19BB0350B1}_is1'
+Installers:
+- Architecture: x86
+  InstallerUrl: https://file1.softsea.com/download/1755275f/daf4282d9530b7d32f8b28411a3c1e1e/Diagnostic_Software/gputemp_setup.exe
```
Comment (Dragon1573) [inline manifests/g/GPUTemp/GPUTemp/1.0/GPUTemp.GPUTemp.installer.yaml:16]:
```suggestion
  InstallerUrl: https://www.softsea.com/download.php?id=836147351
```

**Comment by Dragon1573** [comment]
> When processing this package, the URL to the installer returned a 404 error or was determined to be invalid.
> 
> Please update the URL and hash at your convenience and resubmit your PR.
> 
> Template: msftbot/validationError/urls/404

Manually verified, URL is acceptable.

image

**Review by ItzLevvie — APPROVED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Dragon1573** [comment]
> [!CAUTION]
>
> Close with reason: https://github.com/microsoft/winget-pkgs/labels/Error-Installer-Availability
>
> - See https://github.com/microsoft/winget-pkgs/pull/191713#issuecomment-2479712729

**Pull request closed without merging**